installing_on_kubernetes/_topics/prepare_kubernetes_namespace.md

```diff
@@ -1,4 +1,7 @@
 ## Preparing the Kubernetes namespace
+Prerequisite:
+If running on plain Kubernetes, manual installation of Kafka is required. ManageIQ uses [Strimzi](https://strimzi.io/) and installation steps can be found [here](https://strimzi.io/docs/operators/latest/deploying#deploy-tasks_str). Only the Strimzi cluster operator needs to be deployed, the ManageIQ operator will then make the necessary requests for the resources needed from Strimzi. However if you are using Openshift, the ManageIQ operator will install Strimzi for you and no action is required.
```

Comment:
```suggestion
If you are using OpenShift, our operator will leverage the OpenShift catalog to install Strimzi for you automatically.
If not running on OpenShift, manual installation of Strimzi Kafka is required. [Strimzi](https://strimzi.io/) installation steps can be found [here](https://strimzi.io/docs/operators/latest/deploying#deploy-tasks_str). Only the Strimzi cluster operator needs to be deployed, our operator will then make the necessary requests for the resources needed from Strimzi.
```

Reply:
If the product name is necessary, please use `{{ site.data.product.title_short }}` so that the documentation can be productized.

Reply:
This doc is not used downstream, we confirmed with the docs team